src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -70,14 +73,17 @@
   private final WeakInvalidationListener weakFormatInvalidationListener;
   private final WeakInvalidationListener weakColorPerUserInvalidationListener;
 
+  public ImageView playerMapImage;
   public ImageView playerStatusIndicator;
+  protected Tooltip statusGameTooltip;
+  protected Tooltip gameInfoTooltip;
```

Comment:
java fx tooltips are dangerous in terms of memory leaks. Especially here where content is frequently changed

Reply:
but you seem to take care of it